### PR TITLE
[feat]: Redesign AMP heroimage caption 

### PR DESCRIPTION
**Pull request by shraddha-kesari**
Ref ticket : https://github.com/quintype/ace-planning/issues/877
Amp Pr: https://github.com/quintype/quintype-amp/pull/459

### CODE REVIEW

**Review by cricket-lover — APPROVED** [review]

File: config/publisher.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 asset_host: http://localhost:8080
-sketches_host: https://malibu-advanced-web.qtstage.io
+sketches_host: https://malibu-advanced-web.quintype.io
```
Comment (nandap4790) [inline config/publisher.yml:2]: Is this intentional ?

**Review by nandap4790 — CHANGES_REQUESTED** [review]

File: config/publisher.yml (hunk 1 above)
Comment (nandap4790) [inline config/publisher.yml:2]: is this intentional ?

**Review by nandap4790 — CHANGES_REQUESTED** [review]

File: config/publisher.yml (hunk 1 above)
Comment (shraddha-kesari) [reply config/publisher.yml:2]: Yes. Most of the times staging is down so.

**Review by VeenaYemmiganur — APPROVED** [review]

**Review by nandap4790 — APPROVED** [review]

**Comment by shraddha-kesari** [comment]
This is just updating framework version. Updated framework is been pushed by another PR.

**Pull request closed without merging**